app/controllers/pensions/lifetimeAllowances/PensionLumpSumDetailsController.scala

```diff
@@ -42,40 +42,41 @@ class PensionLumpSumDetailsController @Inject()(implicit val mcc: MessagesContro
                                                 appConfig: AppConfig,
                                                 pensionSessionService: PensionSessionService,
                                                 errorHandler: ErrorHandler,
+                                                formsProvider: FormsProvider,
                                                 clock: Clock) extends FrontendController(mcc) with I18nSupport with SessionHelper with FormUtils {
 
-  def amountForm(isAgent: Boolean): Form[(BigDecimal, BigDecimal)] = TupleAmountForm.amountForm(
-    emptyFieldKey1 = s"lifetimeAllowance.pensionLumpSumDetails.beforeTax.error.noEntry.${if (isAgent) "agent" else "individual"}",
-    wrongFormatKey1 = s"lifetimeAllowance.pensionLumpSumDetails.beforeTax.error.incorrectFormat",
-    exceedsMaxAmountKey1 = s"common.beforeTax.error.overMaximum",
-    emptyFieldKey2 = s"lifetimeAllowance.pensionLumpSumDetails.taxPaid.error.noEntry.${if (isAgent) "agent" else "individual"}",
-    wrongFormatKey2 = s"common.taxPaid.error.incorrectFormat",
-    exceedsMaxAmountKey2 = s"common.taxPaid.error.overMaximum"
-  )
+
 
 
   def show(taxYear: Int): Action[AnyContent] = (authAction andThen taxYearAction(taxYear)).async { implicit request =>
```

Comment:
Any reason not to upgrade to use actionProvider here and to replace getPensionsSessionDataResult with getPensionSessionData